app/models/ability.rb

```diff
@@ -161,6 +161,7 @@ def valkyrie_test_manifest(obj)
   end
 
   def cdl_readable?(obj)
+    obj = Wayfinder.for(obj).parent.present? ? Wayfinder.for(obj).parent : obj if Wayfinder.for(obj).respond_to?(:parent)
```

Comment:
This is a small thing, but I found the ternary confusing here, then realized it was assinging `obj = obj` if the parent wasn't there, which seems unnecessary. Could this check be
`if Wayfinder.for(obj).respond_to?(:parent) && Wayfinder.for(obj).parent.present?` instead?

Reply:
Or just `Wayfinder.for(obj).try(:parent)`

Reply:
Sure, this makes sense.  I'll make the adjustment.